### ISSUE TITLE
exclude_zerolag fails when using two ifos in the multi-ifo workflow

### ISSUE THREAD

**Issue by ahnitz** [gwastro/pycbc#2919]
```
  <statcall error="0" id="stderr">
    <temporary name="/local/condor/execute/dir_17831/ks.err.bB07MY" descriptor="4"/>
    <statinfo mode="0100600" size="563" inode="6442732232" nlink="1" blksize="4096" blocks="8" mtime="2019-09-04T10:31:22+00:00" atime="2019-09-04T10:31:10+00:00" ctime="2019-09-04T10:31:22+00:00" uid="5444" user="ahnitz" gid="5100" group="lvc"/>
    <data>usage: pycbc_multiifo_exclude_zerolag [-h] [--version] [--verbose]
                                      [--statmap-file STATMAP_FILE]
                                      [--other-statmap-files OTHER_STATMAP_FILES [OTHER_STATMAP_FILES ...]]
                                      [--censor-ifar-threshold CENSOR_IFAR_THRESHOLD]
                                      [--veto-window VETO_WINDOW]
                                      [--output-file OUTPUT_FILE]
pycbc_multiifo_exclude_zerolag: error: argument --other-statmap-files: expected at least one argument
```

**Comment by tdent** [comment]
I thought this executable wouldn't actually be used in the 2-ifo workflow.  Is it?

**Comment by ahnitz** [comment]
Sorry, my title was a little less clear than I would have liked. It fails when using 2 ifos in the multi-ifo workflow. The multi-ifo workflow should presumably replace the two-ifo for all cases (even running on just two-ifos).

**Comment by GarethDaviesGW** [comment]
Sorry about this, entirely my fault - I hadn't thought of the possibility of requiring two-ifo implementation on the multiifo pipeline

Will sort this when I can

**Comment by GarethDaviesGW** [comment]
Solved by #2923
